app/src/main/java/com/jerboa/ui/components/home/HomeViewModel.kt

```diff
@@ -77,7 +77,7 @@ class HomeViewModel : ViewModel() {
         }
     }
 
-    fun appendPosts(form: GetPosts) {
+    fun appendPosts(form: GetPosts) =
```

Comment:
Are any of the changes in this file part of the feature?

Reply:
No, sorry, my ide got a bit overzealous

Reply:
However, this one line (80) is important - it causes `appendPosts` to return a `Job` instead of `Unit`, which `SwipeBetweenPosts` then uses to `invokeOnCompletion` after the next page has been loaded. Otherwise though, yes, there are extraneous changes to the file.